compiler/core/src/main.re

```diff
@@ -1,4 +1,6 @@
 open Node;
+open Monad;
```

Comment:
ooh having fun with monad? 😄

Reply:
yeah 😂 this is a total misnomer though, since it only works with Reason's built-in optional type for now